src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -165,6 +165,10 @@ public Mono<Tuple2<Optional<TransactionRefundRequestedEvent>, Either<Transaction
                             );
 
                     if (ClosePaymentRequestV2Dto.OutcomeEnum.OK.equals(closePaymentRequest.getOutcome())) {
+                        final String rrnForClosePayment = getRrnForClosePaymentByAuthorization(
```

Comment:
Modify also rrn saved into [authorization completed event data](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/5c4bce8508293e85f16968f3b1b780f64d891266/src/main/java/it/pagopa/transactions/commands/handlers/TransactionUpdateAuthorizationHandler.java#L77) so that event dispatcher will inherit the correct rrn value to be sent for closure retries